extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/KafkaTuningConfig.java

```diff
@@ -80,6 +86,8 @@ public KafkaTuningConfig(
     this.resetOffsetAutomatically = resetOffsetAutomatically == null
                                     ? DEFAULT_RESET_OFFSET_AUTOMATICALLY
                                     : resetOffsetAutomatically;
+    this.sinkFactory = sinkFactory == null ? defaults.getSinkFactory() : sinkFactory;
+    this.customIndexMerger = customIndexMerger;
```

Comment:
the caller will handle the null customIndexMerger. like in IndexTask: 
```
        ingestionSchema.getTuningConfig().getCustomIndexMerger() != null
        ? ingestionSchema.getTuningConfig().getCustomIndexMerger() : toolbox.getIndexMergerV9()
```
customIndexMerger is a customized IndexMerger which can be null if user don't need it. 
If we set a default IndexMerger implement to customIndexMerger, then I think customIndexMerger should be renamed to `IndexMerger` which should never be null according to the literal meaning